### PR TITLE
Added domain cookie property and autoUnset feature.

### PR DESCRIPTION
**Pull request by ymahnovskyy**

### CODE REVIEW

File: polymer-cookie.html [hunk 1]
```diff
@@ -116,13 +125,21 @@
           type: Boolean,
           value: false,
           reflectToAttribute: true
+        },
+        autoUnset: {
```
Comment (andreasonny83) [inline polymer-cookie.html:129]: Please provide a properties description for consistency with the others and leave a blank space between the previous one.

File: polymer-cookie.html [hunk 2]
```diff
@@ -116,13 +125,21 @@
...
       },
 
       ready: function() {
         if ( this.autoSet ) {
           this.createCookie();
         }
+        if ( this.autoUnset ) {
+          this.eraseCookie();
+        }
       },
```
Comment (andreasonny83) [inline polymer-cookie.html:143]: I would suggest to change this `ready` function in the following way:

```
       ready: function() {
         if ( this.autoUnset ) {
           this.eraseCookie();
         } else if ( this.autoSet ) {
            this.createCookie();
         }
        },
```

**Comment by andreasonny83** [comment]
Good job with that!
I will be happy to merge your pull request as soon as you commit the changes I reviewed.
Also, it would be great if you could update the documentation to include the new functionalities.

Thanks for the collaboration.

**Comment by web-padawan** [comment]
@andreasonny83 would it be OK if I create another PR for `domain` property only?
We need this stuff to create subdomain cookies for keeping auth tokens etc.

**Comment by andreasonny83** [comment]
Sure, looks good to me. Thank you again.

**Comment by web-padawan** [comment]
created #8, please take a look.

**Comment by andreasonny83** [comment]
@web-padawan. Closing this PR now.

**Pull request closed without merging**